app/lib/mqtt_messages_handler.rb

```diff
@@ -30,20 +33,12 @@ def self.handle_readings(topic, message)
     #puts message
   end
 
-  def self.handle_hello(topic, message)
+  def self.handle_hello(device, message)
     payload = {}
-    device_token = self.device_token(topic)
-
-    return if device_token.blank?
-
-    device = Device.find_by(device_token: device_token)
-    if device.present?
-      payload[:device_id] = device.id
-    end
+    payload[:device_id] = device.id
+    payload[:device_token] = device.device_token # TODO: remove after migration
```

Comment:
@pral2a do you remember if we should remove this line now?

Reply:
Yes, we can remove it, finally! That is an important decision as it fixes a critical vulnerability